tests/dappsStaking/tests/test-rpc.js

```diff
@@ -0,0 +1,47 @@
+import { WsProvider, ApiPromise } from '@polkadot/api';
+import plasmTypes from '@plasm/types';
+import { expect } from "chai";
+
+const { plasmDefinitions } = plasmTypes;
+
+const RPC_ENDPOINT = 'wss://astar.api.onfinality.io/public-ws';
+
+async function startAstarNode () {
+	// TODO: start a node with child_process
+	console.log("Starting Astar node...");
+}
+
+describe('Astar RPC', () => {
```

Comment:
I think you should keep the `this` context here:
https://mochajs.org/#arrow-functions